app/models/call_result.rb

```diff
@@ -0,0 +1,24 @@
+class CallResult < ApplicationRecord
+  include Mergeable
+  belongs_to :appointment, optional: true
+  belongs_to :user, optional: true
```

Comment:
Why optional?

Reply:
Good question, I did it out of reflex since all our sync models have optional `user` and `facility` relationships.

This makes sense when the relations are to syncable resources, but I'm not sure why we also do it for users and facilities. I'm okay to make it required and see if it breaks anything.

Reply:
Yup let's be strict from the get go, avoid persisting bad data.

Reply:
We validate this at the db already, so we wouldn't persist bad data. Its just inconsistent, and unnecessary since users aren't a synced from the client.